default-plugins/strider/src/state.rs

```diff
@@ -50,38 +52,88 @@ impl State {
 pub enum FsEntry {
     Dir(PathBuf, usize),
     File(PathBuf, u64),
+    DisplayDir(PathBuf),
 }
 
 impl FsEntry {
     pub fn name(&self) -> String {
         let path = match self {
             FsEntry::Dir(p, _) => p,
             FsEntry::File(p, _) => p,
+            FsEntry::DisplayDir(p) => p,
         };
-        path.file_name().unwrap().to_string_lossy().into_owned()
+        match self {
+            FsEntry::File(..) | FsEntry::Dir(..) => {
+                // only use the filename
+                path.file_name().unwrap().to_string_lossy().into_owned()
+            },
+            FsEntry::DisplayDir(..) => {
+                // use full path, but we need to remove the host part
+                path.to_string_lossy().replace("/host", "")
+            },
+        }
     }
 
     pub fn as_line(&self, width: usize) -> String {
+        let name = self.name();
         let info = match self {
             FsEntry::Dir(_, s) => s.to_string(),
             FsEntry::File(_, s) => pb::convert(*s as f64),
+            FsEntry::DisplayDir(..) => {
+                let current_path = name.clone();
+                FsEntry::display_with_shortened_name_start(current_path, width)
```

Comment:
should just return empty string? just did another run and i'm seeing extra output on the right

Reply:
oh, thanks for pointing this out. I will check this today in the evening.